edd-blockonomics.php

```diff
@@ -57,10 +57,16 @@ public function __construct()
     add_action( 'init',                         array( $this, 'textdomain' ) );
     add_action( 'edd_gateway_blockonomics',         array( $this, 'process_payment' ) );
     add_action( 'init',                         array( $this, 'listener' ) );
-    //	add_action( 'template_redirect',            array( $this, 'process_failed_payment' ) );
```

Comment:
Why is this code commented out? Either should be removed or comment added why this part is in comments. Keeps the code clean

Reply:
The commented code was included in the first commit and has been removed now.